### PR TITLE
Automation

### PR DESCRIPTION
**Pull request by popkinj**
## New Features
- Automating FTP pull and push requests
- Django scheduler installed
- Cron installed and configured

### CODE REVIEW

File: django/requirements.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@ Django==4.0.1
 psycopg2-binary==2.9.3
 djangorestframework==3.13.1
 django-filter==2.4.0
+django-extensions==3.1.3
```
Comment (naomiaro) [inline django/requirements.txt:5]: There's a newer version of extensions already included in requirements.txt

File: django/api/settings.py [hunk 2]
```diff
@@ -45,6 +45,7 @@
     'api.apps.ApiConfig',
     'corsheaders',
     'django_filters',
+    'django_extensions',
```
Comment (naomiaro) [inline django/api/settings.py:48]: These are already installed as an app.

**Review by naomiaro — CHANGES_REQUESTED** [review]

File: django/api/settings.py (hunk 2 above)
Comment (popkinj) [reply django/api/settings.py:48]: Ah nice catch @naomiaro :)

File: django/requirements.txt (hunk 1 above)
Comment (naomiaro) [reply django/requirements.txt:5]: This PR downgraded the version of this package.

File: django/api/settings.py (hunk 2 above)
Comment (naomiaro) [reply django/api/settings.py:48]: This duplication hasn't been removed yet.

Comment (popkinj) [reply django/api/settings.py:48]: Right... Cause I just removed it from [fetch.py](https://github.com/popkinj/itvr/blob/release-0.1.0/django/api/jobs/hourly/fetch.py). Didn't realize there was a duplicate in settings.py as well. Will take care of it.

**Review by naomiaro — APPROVED** [review]

**Pull request merged**